openshift/job-template.yaml

```diff
@@ -0,0 +1,144 @@
+    app: thoth
+    component: package-analyzer
+
+objects:
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: package-analyzer
+      labels:
+        app: thoth
+        component: package-analyzer
+        operator: graph-sync
+        task: package-analyzer
+        mark: cleanup
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          labels:
+            app: thoth
+            component: package-analyzer
+        spec:
+          restartPolicy: Never
+          automountServiceAccountToken: false
+          containers:
+            - name: package-analyzer
+              image: "${THOTH_REGISTRY}/${IMAGE_STREAM_PROJECT_NAME}/package-analyzer"
+              livenessProbe:
+                # Give package analyzer 30 minutes to compute results, kill it if it was not able result anything.
+                # TO BE REVIEWED
+                tcpSocket:
+                  port: 80
+                initialDelaySeconds: 1800
+                failureThreshold: 1
+                periodSeconds: 10
+              env:
+                - name: THOTH_DEPLOYMENT_NAME
+                  valueFrom:
+                    configMapKeyRef:
+                      key: storage-bucket-name
+                      name: thoth
+                - name: THOTH_S3_ENDPOINT_URL
+                  valueFrom:
+                    configMapKeyRef:
+                      key: ceph-host
+                      name: thoth
+                - name: THOTH_CEPH_BUCKET
+                  valueFrom:
+                    configMapKeyRef:
+                      key: ceph-bucket-name
+                      name: thoth
+                - name: THOTH_CEPH_BUCKET_PREFIX
+                  valueFrom:
+                    configMapKeyRef:
+                      key: ceph-bucket-prefix
+                      name: thoth
+                - name: THOTH_CEPH_KEY_ID
+                  valueFrom:
+                    secretKeyRef:
+                      name: thoth
+                      key: ceph-key-id
+                - name: THOTH_CEPH_SECRET_KEY
+                  valueFrom:
+                    secretKeyRef:
+                      key: ceph-secret-key
+                      name: thoth
+                - name: THOTH_PACKAGE_ANALYZER_SUBCOMMAND
+                  value: "python"
+                - name: THOTH_DIGESTS_FETCHER_DEBUG
+                  value: "${THOTH_DIGESTS_FETCHER_DEBUG}"
+                - name: THOTH_DIGESTS_FETCHER_PACKAGE_NAME
+                  value: "${THOTH_DIGESTS_FETCHER_PACKAGE_NAME}"
+                - name: THOTH_DIGESTS_FETCHER_PACKAGE_VERSION
+                  value: "${THOTH_DIGESTS_FETCHER_PACKAGE_VERSION}"
+                - name: THOTH_DIGESTS_FETCHER_INDEX_URL
+                  value: "${THOTH_DIGESTS_FETCHER_INDEX_URL}"
+                - name: THOTH_ANALYZER_OUTPUT
+                  valueFrom:
+                    configMapKeyRef:
+                      key: package-analysis-result
```

Comment:
This needs to be created in thoth-test-core namespace as well

Reply:
Why is it taking this configuration value from a config map? Isn't it supplied from Management API?